README.md

```diff
@@ -388,6 +388,7 @@ Humanize Duration supports the following languages:
 | Romanian             | `ro`     |
 | Russian              | `ru`     |
 | Serbian              | `sr`     |
```

Comment:
Should this be updated to say that it uses Cyrillic? I don't know Serbian so I don't know. Something like "Serbian (Cyrillic)"?